form-validation.js

```diff
@@ -0,0 +1,14 @@
+const form = document.getElementById('form');
+const userEmail = document.getElementById('email');
+const small = userEmail.parentNode.querySelector('small');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  if (userEmail.value !== userEmail.value.toLowerCase()) {
+    small.innerText = 'Form not sent.Please enter email in lower case';
+  } else if (userEmail.value === userEmail.value.toLowerCase()) small.innerText = '';
+});
```

Comment:
- [ ] The requested change by the previous reviewer has not been met, I am still able to submit an email address with uppercase values. Please fix it